pom.xml

```diff
@@ -71,7 +71,7 @@
     <jarsigner.version>3.0.0</jarsigner.version>
 
     <!-- Sloop embedded CLI version for fragment projects -->
-    <sloop.version>10.1.0.77273</sloop.version>
+    <sloop.version>10.1.0.77633</sloop.version>
```

Comment:
This version number is duplicated several times. Would it be possible e.g. to extract it to a property file and refer to it in the `.target` file?

Otherwise, I would add a comment here to remind the next author to update the other files (both `MANIFEST.MF`s + `commons.target`).